src/cli/tools/tofu.ts

```diff
@@ -0,0 +1,63 @@
+import fs from 'node:fs/promises';
+import { join } from 'node:path';
+import { execa } from 'execa';
+import { injectFromHierarchy, injectable } from 'inversify';
+import { BaseInstallService } from '../install-tool/base-install.service';
+
+@injectable()
+@injectFromHierarchy()
+export class tofuInstallService extends BaseInstallService {
```

Comment:
```suggestion
export class TofuInstallService extends BaseInstallService {
```

class names start with upper case